src/components/Settings/preferences/accordions/LanguageAccordion/LanguageAccordion.tsx

```diff
@@ -55,6 +55,16 @@ export const LanguageAccordion: React.FC<LanguageAccordionProps> = ({
         enqueueSnackbar(t('Saved successfully.'), {
           variant: 'success',
         });
+
+        enqueueSnackbar(
```

Comment:
I think this can be removed now

Reply:
Ah you are correct since we don't need it